zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/SparkSQLGeneratorTest.scala

```diff
@@ -388,77 +388,6 @@ class SparkSQLGeneratorTest extends Specification {
           |group by `hour`(t.`create_at`)
           | """.stripMargin.trim)
     }
-    //   28
-    //    "translate a text contain filter and group by geocell 10th" in {
```

Comment:
same here, we want to bookkeep the unimplemented feature. if it hasn't completed, you can leave it empty but leave the title there. 
```
"translate a text contain filter and group by geocell 10th" in {
}
```